aiida_pseudo/groups/family/pseudodojo.py

```diff
@@ -0,0 +1,227 @@
+        # nc-sr-03_pbe
+        PseudoDojoConfiguration('03', 'pbe', 'sr', 'standard', 'none', 'psp8'),
+        PseudoDojoConfiguration('03', 'pbe', 'sr', 'stringent', 'none', 'psp8'),
+        # nc-sr-03_pbesol
+        PseudoDojoConfiguration('03', 'pbesol', 'sr', 'standard', 'none', 'psp8'),
+        PseudoDojoConfiguration('03', 'pbesol', 'sr', 'stringent', 'none', 'psp8'),
+        # nc-sr-03_lda
+        PseudoDojoConfiguration('03', 'lda', 'sr', 'standard', 'none', 'psp8'),
+        PseudoDojoConfiguration('03', 'lda', 'sr', 'stringent', 'none', 'psp8'),
+        # nc-sr-04_pbe_standard
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'standard', 'low', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'standard', 'normal', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'standard', 'high', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'standard', 'none', 'psp8'),
+        # nc-sr-04_pbe_stringent
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'stringent', 'low', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'stringent', 'normal', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'stringent', 'high', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'sr', 'stringent', 'none', 'psp8'),
+        # nc-sr-04_pbesol_standard
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'standard', 'low', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'standard', 'normal', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'standard', 'high', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'standard', 'none', 'psp8'),
+        # nc-sr-04_pbesol_stringent
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'stringent', 'low', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'stringent', 'normal', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'stringent', 'high', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbesol', 'sr', 'stringent', 'none', 'psp8'),
+        # nc-sr-04_lda
+        PseudoDojoConfiguration('04', 'lda', 'sr', 'standard', 'none', 'psp8'),
+        PseudoDojoConfiguration('04', 'lda', 'sr', 'stringent', 'none', 'psp8'),
+        # nc-sr-04-3plus_pbe
+        PseudoDojoConfiguration('04', 'pbe', 'sr3plus', 'standard', 'none', 'psp8'),
+        # nc-fr-04_pbe_standard
+        # PseudoDojoConfiguration('04', 'pbe', 'fr','standard', 'low', 'psp8'),
+        # PseudoDojoConfiguration('04', 'pbe', 'fr','standard', 'normal', 'psp8'),
+        # PseudoDojoConfiguration('04', 'pbe', 'fr','standard', 'high', 'psp8'),
+        PseudoDojoConfiguration('04', 'pbe', 'fr', 'standard', 'none', 'psp8'),
+        # nc-fr-04_pbe_stringent
+        # PseudoDojoConfiguration('04', 'pbe', 'fr', 'stringent', 'low', 'psp8'),
+        # PseudoDojoConfiguration('04', 'pbe', 'fr', 'stringent', 'normal', 'psp8'),
+        # PseudoDojoConfiguration('04', 'pbe', 'fr', 'stringent', 'high', 'psp8'),
```

Comment:
What does it mean for these to be commented out?

Reply:
They are commented out because the suggested cutoff `djson` files are bugged with either missing elements or md5 hashes that do not match the files downloaded from the PseudoDojo website.

Reply:
They are multiple ways to solve this: using a high ecut like I mentioned above or using the corresponding SR hint values. Indeed I believe the FR psp are generated with the same values as the SR one but just solving the Dirac equation instead of Schrodinger. Therefore one would expect the same or close values for energy cutoff.

Reply:
would it be possible to get the djson files fixed on the pseudo dojo website @sponce24 ? I imagine this would be in their interest regardless of this plugin.

Reply:
We're working with the maintainers to get this worked out; it may take some time.